apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.tsx

```
@@ -259,6 +245,69 @@ export function AssessmentStats({ row }: { row: AssessmentStatsRow }) {
+        type: 'instructor',
+        page: 'instance_admin',
+        subPage: 'assessments',
+      },
+      options: {
+        fullWidth: true,
+      },
+      content: (
+        <Fragment>
+          <div
+            // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)